README.md

```
@@ -158,33 +162,70 @@ docker-image-scripts/
+        description: 3
 ```
-Date,Description,Amount,Category
-2023-01-15,Grocery shopping,125.50,Groceries
-2023-01-20,Monthly rent,1200.00,Housing
-2023-01-25,Internet bill,65.00,Utilities
+
+### Standardised Internal Format (Scheduler)
+
+The scheduler reads CSV files in this fixed format (written by the Import UI after review):
+
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)